File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.py
Num = [i for i in range (1,31)]    #1부터 30까지의 리스트 생성

for i in range(28):
    n = int(input())
    Num.remove(n)        #remove 함수를 이용한 소거
    
print(min(Num))
print(max(Num))

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.Collections;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        ArrayList<Integer> list = new ArrayList<>();
        for (int i = 1; i <= 30; i++) {
            list.add(i);
        }

        for (int j = 1; j <= 28; j++) {
            int T = Integer.parseInt(br.readLine());
            list.remove(Integer.valueOf(T));    // T를 값으로 인식하게 하기 위해 Integer 객체로 변환시켜야함!
        }

        int min = Collections.min(list);
        int max = Collections.max(list);
        System.out.println(min);
        System.out.println(max);
    }
}
